picoscopeimpl/include/picoscopeimpl/status_string.h

```diff
@@ -3,7 +3,15 @@
 #include <PicoStatus.h>
 #include <string>
 
-inline std::string ps4000a_status_to_string(PICO_STATUS status)
+#include <fmt/format.h>
+
+// Helper functions to get readable messages from PICO_STATUS values. Assumes
+// that PICO_STATUS is identical for all psXXXX implementations, which is 
+// currently the case.
+
+namespace gr::picoscope::detail {
```

Comment:
Why to have the string ` ::detail `in the namespace ? If a suffix is required, e.g.` gr::picoscope::status` imo would fit better.